File: client/src/model/AdditionalWork.ts
import Employee from './Employee'

interface IAdditionalWork {
	id: number
	employee: Employee
	valuation: number
	metalOrder: number
	drawingsCompleted: number
	drawingsCheck: number
}

class AdditionalWork {
	id: number
	employee: Employee
	valuation: number
	metalOrder: number
	drawingsCompleted: number
	drawingsCheck: number

	constructor(obj?: IAdditionalWork) {
		this.id = (obj && obj.id) || 0
		this.employee = (obj && obj.employee) || null
		this.valuation = (obj && obj.valuation) || 0
		this.metalOrder = (obj && obj.metalOrder) || 0
		this.drawingsCompleted = (obj && obj.drawingsCompleted) || 0
		this.drawingsCheck = (obj && obj.drawingsCheck) || 0
	}
}

export default AdditionalWork

File: server/DocumentsKM/Models/AdditionalWork.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DocumentsKM.Models
{
    // Дополнительные работы
    public class AdditionalWork
    {
        [Key]
        public Int32 Id { get; set; }

        // Марка
        [Required]
        public virtual Mark Mark { get; set; }

        // Исполнитель
        [Required]
        public virtual Employee Employee { get; set; }

        // Расчет
        [Required]
        [Range(0, 65535, ErrorMessage = "Value should be greater than or equal to 0")]
        public Int16 Valuation { get; set; }

        // Заказ металла
        [Required]
        [Range(0, 65535, ErrorMessage = "Value should be greater than or equal to 0")]
        public Int16 MetalOrder { get; set; }
    }
}
